CMakeLists.txt

```diff
@@ -175,11 +175,13 @@ if (USE_TLS AND USE_OPEN_SSL)
     set(CMAKE_INCLUDE_PATH ${CMAKE_INCLUDE_PATH} /usr/local/opt/openssl/include)
   endif()
 
-  find_package(OpenSSL REQUIRED)
+  if(NOT OPENSSL_FOUND)
+    find_package(OpenSSL REQUIRED)
+  endif()
   add_definitions(${OPENSSL_DEFINITIONS})
   message(STATUS "OpenSSL: " ${OPENSSL_VERSION})
   include_directories(${OPENSSL_INCLUDE_DIR})
-  target_link_libraries(ixwebsocket ${OPENSSL_LIBRARIES})
+  target_link_libraries(ixwebsocket ${OPENSSL_LIBRARIES} $<$<PLATFORM_ID:Windows>:Shlwapi.lib>)
```

Comment:
Can you add this target_link_libraries inside an if (WIN32)

Reply:
I moved `shlwapi` into a WIN32 block.